services/blockchain-connector/tests/unit/fs.test.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { resolve, dirname } = require('path');
+
+const {
+	exists,
+	extractTarBall,
+	mkdir,
+	read,
+	rm,
+} = require('../../shared/utils/fs');
+
+const { genesisBlock } = require('../constants/blocks');
+
+describe('Unit tests for filesystem utilities', () => {
+	const dirPath = resolve(`${dirname(__dirname)}/testDir`);
+	const filePath = resolve(`${process.cwd()}/tests/constants/genesis_block.json`);
+	const tarFilePath = resolve(`${process.cwd()}/tests/constants/genesis_block.json.tar.gz`);
```

Comment:
Isn't it better to resolve path from `__dirname` as path from process depends on the script execution directory?